mule-extensions-api/src/main/java/org/mule/runtime/extension/api/introspection/declaration/type/annotation/DisplayTypeAnnotation.java

```diff
@@ -62,8 +64,8 @@ public String getSummary() {
   public boolean equals(Object obj) {
     if (obj instanceof DisplayTypeAnnotation) {
       DisplayTypeAnnotation displayTypeAnnotation = ((DisplayTypeAnnotation) obj);
-      return displayTypeAnnotation.getDisplayName().equals(this.getDisplayName())
-          && displayTypeAnnotation.getSummary().equals(this.getSummary());
+      return Objects.equals(displayTypeAnnotation.getDisplayName(), this.getDisplayName()) &&
```

Comment:
wasn't the whole point to delegate in the model? and you're still missing the hashCode